easybuild/easyconfigs/n/NCL/NCL-6.6.2-foss-2020b.eb

```diff
@@ -0,0 +1,56 @@
+name = 'NCL'
+version = '6.6.2'
+
+homepage = 'https://www.ncl.ucar.edu'
+description = "NCL is an interpreted language designed specifically for scientific data analysis and visualization."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {
+    'cstd': 'c99', 'openmp': True, 'pic': True,
+    'extra_fflags': '-fallow-invalid-boz -fallow-argument-mismatch',
+}
+
+source_urls = ['https://github.com/NCAR/ncl/archive/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    'NCL-6.4.0_fix-types.patch',
+    'NCL-6.6.2_header_and_operands.patch',
+    'NCL-6.6.2_multiple_definitions.patch',
+]
+checksums = [
+    'cad4ee47fbb744269146e64298f9efa206bc03e7b86671e9729d8986bb4bc30e',  # 6.6.2.tar.gz
+    'f6dfaf95e5de9045745e122cb44f9c035f81fab92f5892991ddfe93945891c8f',  # NCL-6.4.0_fix-types.patch
+    '5cb62d9a37de8af485b94487692f7e739ca5eb103a6b447e0290a68a6c094808',  # NCL-6.6.2_header_and_operands.patch
+    'e21503dc46fe6fd360ba7ca4897aa93bf9ec2961bc48634ca46de582710448f7',  # NCL-6.6.2_multiple_definitions.patch
+]
+
+builddependencies = [
+    ('makedepend', '1.0.6'),
+    ('Bison', '3.7.1'),
+    ('flex', '2.6.4'),
+]
+dependencies = [
+    ('cURL', '7.72.0'),
+    ('JasPer', '2.0.24'),
+    ('g2lib', '3.2.0'),
+    ('g2clib', '1.6.3'),
+    ('HDF', '4.2.15'),
+    ('HDF5', '1.10.7'),
+    ('netCDF', '4.7.4'),
+    ('netCDF-Fortran', '4.5.3'),
+    ('Szip', '2.1.1'),
+    ('freetype', '2.10.3'),
+    ('zlib', '1.2.11'),
+    # ('GDAL', '3.9.1'),
+    ('UDUNITS', '2.2.26'),
+    ('ESMF', '8.0.1'),
+    ('bzip2', '1.0.8'),
+    ('cairo', '1.16.0'),
+    ('libiconv', '1.16'),
+    ('GSL', '2.6'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.5'),
+    # ('pixman', '0.40.0'),
```

Comment:
`pixman` is a dependency of `cairo`, so wouldn't be needed here in any case - delete commented line?